en/0x13-V4-Package_Management.md

```diff
@@ -35,4 +35,4 @@ compromise in the software supply chain.
 | **4.15** | Package repository enforces use of TLS for all interactions | ✓ | ✓ | ✓ |
 | **4.16** | Package manager validates TLS certificate chain to repository and fails securely when validation fails | ✓ | ✓ | ✓ |
 | **4.17** | Package repository requires and/or performs static code analysis prior to publishing a component and makes results available for others to consume | | | ✓ |
-| **4.18** | Package manager does not execute code | ✓ | ✓ | ✓ |
+| **4.18** | Package manager does not execute component code | ✓ | ✓ | ✓ |
```

Comment:
Fine.
Add new control after this that states:
"Package manager documents package installation in machine-readable form"